reana/cli.py

```diff
@@ -1675,9 +1768,9 @@ def cluster_undeploy():  # noqa: D301
 @click.option(
     "-m",
     "--mount",
-    is_flag=True,
-    default=False,
-    help="Should we mount /var/reana from host?"
+    "mounts",
+    multiple=True,
+    help="Which local directories would be mounted in the Kind nodes? local_path=reana_system_path",
```

Comment:
The syntax is actually `local_path:cluster_node_path` using `:`, not `=`.

(probably ditto for job mounts)